Makefile

```diff
@@ -0,0 +1,20 @@
+install:
+	cargo install cross
+
+build:
+	make build-linux
+	make build-mac
+	make shasum
+
+build-linux:
+	@echo 'Building for Linux... 🐧'
+	cross build --release --target=x86_64-unknown-linux-musl
+	mkdir -p target/release-archives && tar -C target/x86_64-unknown-linux-musl/release -czf target/release-archives/sic-linux.tar.gz sic
+
+build-mac:
+	@echo 'Building for MacOS... 🍏'
+	cross build --release --target=x86_64-apple-darwin
```

Comment:
cross doesn't seem to [support](https://github.com/rust-embedded/cross#supported-targets) this target; any idea whether I am doing something wrong? :slightly_smiling_face:

Reply:
Does it work when you delete the whole `target` folder first? 🤔

Reply:
I tried both manually removing the target folder and `cargo clean` which presumably does the same thing; unfortunately I get the same result :cry:

Reply:
Ah yeah right, this works on MacOS but not on Ubuntu 🤦‍♂️
Hmm I think back in the day this also worked for Ubuntu 😬

Reply:
Well, I think I will add a comment that this only works on MacOS currently, upload my MacOS archive on that formula repo and publish it that way 😊